### PR TITLE
Update bria from 6.0.1_102001 to 6.0.3_102645

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Is it a walled build now? Not seeing a public download page.

**Comment by suschizu** [comment]
you are right - the download seems to be walled now (I'm not sure if it was also walled last time)
The redirect showed a new url - so I changed it 😬

**Comment by vitorgalvao** [comment]
> I'm not sure if it was also walled last time

It only counts as walled if it’s on a different domain than the homepage, so even if the previous was behind a login, it wasn’t walled.

File: Casks/bria.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 cask 'bria' do
-  version '6.0.1_102001'
-  sha256 'd327f8518f8c4aa16e67f3c557fe4985c158d448f889f87f0bf1858a6ae95ea0'
+  version '6.0.3_102645'
+  sha256 '9730d408fabecc1cfb6e525d64973ea1cb6b7819e7f7f22863740557a50663a6'
 
-  url "https://secure.counterpath.com/Downloads/Bria_#{version}.dmg"
+  # s3.amazonaws.com was verified as official when first introduced to the cask
+  url "https://counterpath.s3.amazonaws.com/downloads/Bria_#{version}.dmg"
   appcast 'https://www.macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://solo.softphone.com/downloads/bria-for-mac'
```
Comment (vitorgalvao) [inline Casks/bria.rb]: No longer working.

Comment (suschizu) [reply Casks/bria.rb]: ok crap - so just wait if the link will work again?

Comment (vitorgalvao) [reply Casks/bria.rb]: If there’s nothing better, perhaps it’s best to wait a few days and see.

**Comment by core-code** [comment]
how about this for an appcast?

https://www.macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://www.counterpath.com/Bria6forMac

**Comment by core-code** [comment]
try this for the checksum

`e1acfe6f624f276f318aa77504b3b11192b2c31acf89ed0b16c55618a1710c50`

**Comment by core-code** [comment]
is there anything still blocking this?

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**